SkEditor/Controls/Docs/DocElementControl.axaml.cs

```diff
@@ -0,0 +1,446 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Controls.Primitives;
+using Avalonia.Interactivity;
+using Avalonia.Layout;
+using Avalonia.Media;
+using Avalonia.Styling;
+using AvaloniaEdit;
+using AvaloniaEdit.Highlighting;
+using FluentAvalonia.UI.Controls;
+using SkEditor.API;
+using SkEditor.Utilities.Docs;
+using SkEditor.Utilities.Docs.Local;
+using SkEditor.Utilities.Syntax;
+using SkEditor.Views;
+
+namespace SkEditor.Controls.Docs;
+
+public partial class DocElementControl : UserControl
+{
+    private bool _hasLoadedExamples;
+    private DocumentationControl _documentationControl;
+    private IDocumentationEntry _entry;
+    
+    public DocElementControl(IDocumentationEntry entry, DocumentationControl documentationControl)
```

Comment:
The constructor is soo long... I'd split it up into some methods.
The entire class is pretty long too, so it's worth to consider if it shouldn't be split into several classes.